ui/app/components/auth-saml.js

```diff
@@ -17,83 +19,79 @@ const ERROR_MISSING_PARAMS =
   'The callback from the provider did not supply all of the required parameters. Please click Sign In to try again. If the problem persists, you may want to contact your administrator.';
 export { ERROR_WINDOW_CLOSED, ERROR_MISSING_PARAMS };
 
-export default Component.extend({
-  store: service(),
-  featureFlagService: service('featureFlag'),
+export default class AuthSaml extends Component {
+  @service store;
+  @service featureFlag;
```

Comment:
I am keeping this here in case I'm missing something, but I don't see this service being used anywhere. It is called a different way this this.featureFlag (or before this.featureFlagService)?